libshaderc/include/shaderc.h

```diff
@@ -63,6 +63,15 @@ typedef enum {
   shaderc_profile_es,
 } shaderc_profile;
 
+// Used in the result module (shaderc_spv_module) to tell the type of errors
+// generated during compilation.
```

Comment:
Should be shader_compilation_status_* instead of shader_compilation_result_*

A "result" sounds like a more heavyweight thing, as if it included the resulting binary.

Precedent is OpenCL CL_PROGRAM_BUILD_STATUS query from clGetProgramBuildInfo.
Also, COMPILE_STATUS query from GL's GetShaderiv query.